client/src/router/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { createBrowserRouter } from "react-router-dom";
+import { TodoAppContent } from "@/todo/TodoAppContent";
+import React from "react";
+import IndexContent from "../index/IndexContent";
+
+// routes are declared here
+const router = createBrowserRouter([
+	// there is an option where you can declare routes by using components, but for now I decided to use good old json object format.
+	{
+		path: "/",
+		element: <IndexContent />
+	},
+	{
+		path: "/todo",
+		element: <TodoAppContent />
+	}
+])
+
+export default router;
```

Comment:
I think it's more common to return a component that wraps the router entirely from this file. So instead of `export default router`, I'd do 

```ts
export const AppRouter= () => <RouterProvider route={route} />;
```

That way the entire mechanism we are using for routing (react-router) is hidden from the file importing this.

Also, small thing, but generally "best practice" is to _not_ use default exports